.vscode/launch.json

```diff
@@ -15,13 +15,21 @@
     },
     {
       "name": "Next.js: debug full stack",
-      "type": "node-terminal",
+      "type": "node",
       "request": "launch",
-      "command": "yarn dev",
```

Comment:
Sorry, I'll look into these a bit more. I just copied the default settings from NextJS because the existing setting were not working for me. I could not get breakpoints attach

Reply:
Reverted this change